frontends/qt/main.cpp

```diff
@@ -111,12 +121,60 @@ class WebEnginePage : public QWebEnginePage {
     }
 };
 
+// Custom scheme handler so we can force mime type resolution manually.
+// Without this, on linux the local mimetype database can mess with the mimetype resolution.
+// Programs like monodevelop or Steam/Proton install weird mime types making for example our
+// .js or .html files be served with the wrogn mimetype, resulting in the webengine displaying a
+// blank page only.
+class SchemeHandler : public QWebEngineUrlSchemeHandler {
+public:
+    // Similar like the built-in qrc scheme handler, but with hardcoded mime-types.
+    // https://github.com/qt/qtwebengine/blob/v6.2.4/src/core/net/qrc_url_scheme_handler.cpp
+    void requestStarted(QWebEngineUrlRequestJob *request) override {
+        QByteArray requestMethod = request->requestMethod();
+        if (requestMethod != "GET") {
+            request->fail(QWebEngineUrlRequestJob::RequestDenied);
+            return;
+        }
+        QUrl url = request->requestUrl();
+        QString path = url.path();
+
+        QMimeDatabase mimeDb;
+
+        QMimeType mimeType = mimeDb.mimeTypeForFile(path);
+
+        QString hardcodedMimeType;
+        if (path.endsWith(".html")) {
+            hardcodedMimeType = "text/html";
+        } else if (path.endsWith(".js")) {
+            hardcodedMimeType = "application/javascript";
+        } else if (path.endsWith(".css")) {
+            hardcodedMimeType = "text/css";
+        } else if (path.endsWith(".svg")) {
+            hardcodedMimeType = "image/svg+xml";
+        } else {
```

Comment:
The only ones missing as far as I see are .webm (for the password input video) and .png

Reply:
Pls provide the mimetypes.

The only practical isues we had were with .html and .js, the others I put just in case but are not fixing any known bugs, so it's not important to add all of them right away.

Reply:
in chromium
- ".png" shows "image/png" 
- ".webm" uses "video/webm"

I didn't find any other endings in `frontends/web/build`